index.bs

```diff
@@ -972,6 +987,7 @@ Note: `dispatch()` itself provides no signal that graph execution has completed.
     1. If [=validating tensors with descriptors=] given |inputs| and |graph|.{{MLGraph/[[inputDescriptors]]}} returns false, then [=exception/throw=] a {{TypeError}}.
     1. If [=validating tensors with descriptors=] given |outputs| and |graph|.{{MLGraph/[[outputDescriptors]]}} returns false, then [=exception/throw=] a {{TypeError}}.
     1. Enqueue the following steps to |graph|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}:
+        1. If |graph|.{{MLGraph/[[context]]}} is no longer available to fulfill requests, then run the [=context lost=] steps for it.
```

Comment:
Use "abort when" here as well?

Reply:
Done. 

The formulation I'm using of "Enqueue... which abort when... " is a shortcut for:

1. Enqueue the following steps to ...
     1. Run these steps, but abort when ...
         1. do it
         2. do more of it
     1. If aborted, ....
 
Should I expand it out?

Reply:
The expanded form is probably better, because right now the indentation of the "if aborted" steps doesn't make it look like they are also supposed to be running in parallel.

Reply:
... and then we need to figure out how to represent "lost" over on the context's timeline - right now the concept sits on the ML task runner.

Reply:
Done in 88e8d58 - I left "is lost" alone for now though.